packages/js/src/ui/components/Renderer.tsx

```diff
@@ -67,33 +68,40 @@ type RendererProps = {
   preferencesFilter?: PreferencesFilter;
   routerPush?: RouterPush;
   novu?: Novu;
+  containerElement?: Node | null | undefined;
```

Comment:
the new prop that defines where the Inbox and its styles will be embedded; the name TBD;